src/components/RoundImageRow/RoundImageRow.js

```diff
@@ -26,7 +26,7 @@ export default function RoundImageRow ({ imageUrls = [], className, vertical, ca
       styleName='image'
       style={zIndexStyle(i)} />)
 
-  const plus = <div styleName='plus' key='plus' style={zIndexStyle(imageUrls.length)} >
+  const plus = <div styleName={onProfile ? 'plus-profile' : 'plus-default'} key='plus' style={zIndexStyle(imageUrls.length)} >
```

Comment:
instead of having onProfile its more flexible to just allow for passing in a className/styleName to this component and then the styles can be specified in the parent component

Reply:
@tibetsprague so you are thinking of passing two classNames to this component, since one goes for the parent div and then another is used on the internal "plus" div?

Reply:
You could create a `template` or `styles` prop that was an hash with classnames under the `plus` and `container` keys (or similar). With something like that you can optionally use a named import of styles.scss as `styles` in the parent component and directly send the whole styles hash table to the component to work with (e.g. `<RoundImageRow ... styles={styles} />`)

You could also leave the className applying over the container element and combining with the optional template provided style... 

\* I've always wished far leaf components to be style-less as much as possible, but it's often not so easy to do so. I like the template pattern I'm suggesting, but other devs I've ran that past often find it confusing or don't feel the extra complication is warranted.

Reply:
I think this is extra complexity not needed for now since there are only two variants, one where the plus circle is on the next line, and one where the plus circle is inline (used on the profile page for now). I've changed the boolean to be `inline` rather than `onProfile` to reflect this.

Reply:
great